gammapy/makers/background/tests/test_reflected.py

```diff
@@ -114,19 +110,18 @@ def test_find_reflected_regions(
     )
     finder.region = on_ellipse_annulus
     finder.reference_map = None
-    finder.run()
-    regions = finder.reflected_regions
+    regions = finder.run()
     assert len(regions) == 5
 
 
 center = SkyCoord(0.5, 0.0, unit="deg")
 other_region_finder_param = [
     (RectangleSkyRegion(center, 0.5 * u.deg, 0.5 * u.deg, angle=0 * u.deg), 3),
     (RectangleSkyRegion(center, 0.5 * u.deg, 1 * u.deg, angle=0 * u.deg), 1),
-    (RectangleSkyRegion(center, 0.5 * u.deg, 1 * u.deg, angle=90 * u.deg), 0),
+    (RectangleSkyRegion(center, 0.5 * u.deg, 1 * u.deg, angle=90 * u.deg), 1),
     (EllipseSkyRegion(center, 0.1 * u.deg, 1 * u.deg, angle=0 * u.deg), 2),
     (EllipseSkyRegion(center, 0.1 * u.deg, 1 * u.deg, angle=60 * u.deg), 3),
-    (EllipseSkyRegion(center, 0.1 * u.deg, 1 * u.deg, angle=90 * u.deg), 0),
+    (EllipseSkyRegion(center, 0.1 * u.deg, 1 * u.deg, angle=90 * u.deg), 2),
```

Comment:
Why the change here?

Reply:
I will try to understand the change and comment again...

Reply:
# Before:
![Screenshot 2021-07-14 at 10 14 50](https://user-images.githubusercontent.com/3715928/125587867-2c51fd75-1c8c-493c-a50a-f4b663df0a07.png)
# After:
![Screenshot 2021-07-14 at 10 12 35](https://user-images.githubusercontent.com/3715928/125587884-12209d20-0748-4749-ae40-e1d5e6f86927.png)

I think placing the regions for the last column is within the numerical precision of the algorithm, which depends on the bin  size of the tangential WCS. I'm not 100% why the behaviour changed, but result still seems OK.

Reply:
In general one could change to a less sensitive test, but maybe it's better to keep the more sensitive one and check the results once by hand.